doc/architecture/frontend.md

```diff
@@ -0,0 +1,57 @@
+# Frontend Architecture
+
+## Structure
+
+- `__mocks__/`: Contains mocked modules for tests
+- `assets/`: Contains asset files like images
+- `bindings/`: Contains the generated API bindings
+- `components/`: Contains components that are shared across features
```

Comment:
Write that they need to be part of Storybook. Describe minimum requirements for documentation.